tests/CSharpLanguageServer.Tests/TestData/testSemanticToken/Project/Program.cs

```diff
@@ -0,0 +1,4 @@
+var a = $"""
+test for
+multiline token
+""";
```

Comment:
I would prefer there to be more constructs if we are testing SemanticToken requests, i.e. a class, a couple of methods, namespace decl, etc. But that may be outside of the scope of this PR–feel free to leave it as is.